src/Core/Commander/Providers/Fetcher.js

```diff
@@ -2,6 +2,8 @@ import { TextureLoader } from 'three';
 
 const textureLoader = new TextureLoader();
 
+textureLoader.setCrossOrigin('');
```

Comment:
If I'm not mistaken, `setCrossOrigin` [forwards to](https://github.com/mrdoob/three.js/blob/dev/src/loaders/ImageLoader.js#L71) img tag [crossOrigin property](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-crossorigin). Possible values are `anonymous` and `with-credentials`. (empty is equivalent to `anonymous` if I'm not mistaken).

I guess we probably want `with-credentials`? Just in case there is a http auth on the wms/wmts/whatever server...